src/components/organisms/PageHeader/HelpMenu.tsx

```diff
@@ -43,6 +60,18 @@ const HelpMenu = () => {
         </Row>
       </MenuItem>
 
+      <MenuItem onClick={onClickReleaseNotes} key="releasenotes" style={{paddingRight: '20px'}}>
```

Comment:
I can see that this was not introduced by you, it seems that all the menu items have a right padding of 20px, maybe we can move this to a styled file?